kolibri/plugins/device/assets/src/views/WelcomeModal.vue

```diff
@@ -70,6 +80,14 @@
         message: `The learner reports, lessons, and quizzes in '{facilityName}' will not display properly until you import the resources associated with them.`,
         context: 'Welcome message for user indicating that they need to import resources.',
       },
+      posLODWelcomeMessage1: {
+        message: 'The first thing you should do is import some channels to this device',
+        context: 'Welcome message for user which appears after provisioning a Learner Only Device.',
+      },
+      posLODWelcomeMessage2: {
+        message: `The user reports, lessons, and quizzes in  '{facilityName}' will not display properly until you import the resources associated with them.`,
+        context: 'Welcome message for user indicating that they need to import resources.',
+      },
```

Comment:
since message IDs are shown to translators, these should be spelled out using the user-facing terminology:

```suggestion
      learnOnlyDeviceWelcomeMessage1: {
        message: 'The first thing you should do is import some channels to this device',
        context: 'Welcome message for user which appears after provisioning a Learner Only Device.',
      },
      learnOnlyDeviceWelcomeMessage2: {
        message: `The user reports, lessons, and quizzes in  '{facilityName}' will not display properly until you import the resources associated with them.`,
        context: 'Welcome message for user indicating that they need to import resources.',
      },
```

Reply:
Done